app/src/main/AndroidManifest.xml

```diff
@@ -27,7 +28,8 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
```

Comment:
these changes should be excluded from this PR, which hasn't the relationship to the topic.

Reply:
Done.